### ISSUE TITLE
('child_process').exec fails in Travis

### ISSUE THREAD

**Issue by inwardmovement** [nodejs/help#2210]
* **Version**: v10.16.3
* **Platform**: Windows 64
* **Subsystem**: Child Process

`('child_process').exec` fails in Travis ([log](https://travis-ci.org/inwardmovement/mettaconseilpro/jobs/590748390#L504)) (it's ok locally), can someone help please?

```
Error: exited with error code: 255
    at ChildProcess.onexit (/home/travis/build/inwardmovement/mettaconseilpro/node_modules/end-of-stream/index.js:39:36)
    at ChildProcess.emit (events.js:198:13)
    at ChildProcess.EventEmitter.emit (domain.js:466:23)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:248:12)
```

**Comment by Hakerh400** [comment]
You are missing two global dependencies. Replace

```
- npm install -g gulp-cli
```

with

```
- npm install -g gulp-cli postcss-cli autoprefixer
```

in `.travis.yml`

Side note: never use `console.log()` inside the `child_process.exec` because Gulp hides the output. If you want to log the output, either use `spawn` or do not return the result of the `exec` call.

**Comment by inwardmovement** [comment]
Oh, my bad, I forgot Hugo needs them as global dependencies, you were right, thanks very much!

For the `console.log()` thing, I took it from the example in the [gulp exec package](https://www.npmjs.com/package/gulp-exec).

So you mean
```
exec('hugo', function (err, stdout, stderr) {
    console.log(stdout);
    console.log(stderr);
    fetch(err);
```

is ok but
```
return exec('hugo', function (err, stdout, stderr) {
    console.log(stdout);
    console.log(stderr);
    fetch(err);
```

is not? It seems ok for Travis too, so I removed the `return`, but I don't understand all the stakes, I hope it won't mess another thing up.
